SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1

```diff
@@ -85,6 +85,9 @@ function Get-Secret
 
     # TODO error handling
 
+    # -eq $true is used since the value will be null before Preview 5, which is equivalent to -verbose:$true
+    $VerboseFlag = @{Verbose = $AdditionalParameters.Verbose -eq $true } 
```

Comment:
What about:

```pwsh
if ($AdditionalParameters.Verbose) {
    $VerbosePreference = "Continue"
}
```

Then you shouldn't need the `@VerboseFlag` I don't think.

Reply:
Indeed. Fixed.